INSTALL.md

```diff
@@ -1,16 +1,35 @@
 # Install
 
+## Quick Start
+
+If you just want to try out the astrobee simulator, you can use one of the docker images in the [Docker Hub](https://hub.docker.com/r/astrobee/astrobee). Make sure you have docker installed in your ubuntu system flolowing the [installation instructions](https://docs.docker.com/engine/install/ubuntu/) and [post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/).
+Given that Docker is installed, just run:
+
+    ./scripts/docker/run.sh
```

Comment:
I think you are assuming that the user has checked out the main astrobee repo, and this is a path in that repo. But the user might first find the github.io documentation tree, in which case when they read this, they aren't assuming the context of the astrobee repo and might get confused.